tf2onnx/utils.py

```diff
@@ -154,16 +171,15 @@ def get_tf_tensor_data(tensor):
         data = tensor.int64_val
     elif tensor.bool_val:
         data = tensor.bool_val
-    elif tensor.dtype == tf.int32:
-        data = [0]
-    elif tensor.dtype == tf.int64:
-        data = [0]
-    elif tensor.dtype == tf.float32:
-        data = [0.]
-    elif tensor.dtype == tf.float16:
-        data = [0]
     elif tensor.string_val:
         data = tensor.string_val
+    elif tensor.dtype in [
+            tf.int32,
```

Comment:
please have a comprehensive testing for this change locally.

Reply:
all tests including internal models are passed.